src/20250721_Multi_InterestRateUpdateWETHAndWstETHEthereum/AaveV3EthereumLido_InterestRateUpdateWETHAndWstETHEthereum_20250721.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3EthereumLidoAssets} from 'aave-address-book/AaveV3EthereumLido.sol';
+import {AaveV3PayloadEthereumLido} from 'aave-helpers/src/v3-config-engine/AaveV3PayloadEthereumLido.sol';
+import {EngineFlags} from 'aave-v3-origin/contracts/extensions/v3-config-engine/EngineFlags.sol';
+import {IAaveV3ConfigEngine} from 'aave-v3-origin/contracts/extensions/v3-config-engine/IAaveV3ConfigEngine.sol';
+
+/**
+ * @title Interest Rate Update - WETH and wstETH Ethereum
```

Comment:
We only update wstETH so the title should have been only with "wstETH" but renaing this means rename every file so let's keep it like this